YoCode/GitCheck.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+using System.IO;
+
+namespace YoCode
+{
+    public class GitCheck
+    {
+        //needs to be fixed
+        public string REPOSITORY_PATH = @"C:\Users\ukmzil\source\repos\Tests Sent by People\Real\simon-jones";
```

Comment:
For all files uses same path, cannot check submission with name that is not simon-jones